### PR TITLE
DP-24950 expose collections on /admin/ma-dash/documents

### PR DESCRIPTION
**Pull request by beto-aveiga**
**Description:**
- Exposing collections on /admin/ma-dash/documents
- Adding collections filter to /admin/ma-dash/documents and its "export to CSV" display.
- Adding the collections field on the CSV file.

**Jira:** (Skip unless you are MA staff)
DP-24950

**To Test:**
- [ ] Visit /admin/ma-dash/documents
- [ ] Select a collection.
- [ ] TIP: Narrow your search if needed using the title filter.
- [ ] Filter the view.
- [ ] Open in a new tab the link for CSV on the bottom of the page.
- [ ] Compare the results, must be the same results.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: conf/drupal/config/views.view.all_documents.yml [hunk 1]
```diff
@@ -1126,6 +1128,65 @@ display:
...
+          group_type: group
+          admin_label: ''
+          plugin_id: taxonomy_index_tid
+          operator: or
+          value: {  }
+          group: 1
+          exposed: true
+          expose:
+            operator_id: field_collections_target_id_op
+            label: 'Collections (field_collections)'
```
Comment (joeg8612) [inline conf/drupal/config/views.view.all_documents.yml]: let's just get rid of the field name in parentheses here. We don't do this for any other field. Then

File: conf/drupal/config/views.view.all_documents.yml [hunk 2]
```diff
@@ -3170,6 +3297,65 @@ display:
...
```
Comment (joeg8612) [inline conf/drupal/config/views.view.all_documents.yml]: Remove field name that is in parentheses.

Comment (arthurbaghdas) [reply conf/drupal/config/views.view.all_documents.yml]: All set

File: conf/drupal/config/views.view.all_documents.yml (hunk 1 above)
Comment (arthurbaghdas) [reply conf/drupal/config/views.view.all_documents.yml]: All set

**Review by arthurbaghdas — APPROVED** [review]

**Pull request merged**